sentences/fr/_common.yaml

```diff
@@ -7,9 +7,59 @@ responses:
     no_device_class: "{{ area }} ne contient pas de {{ device_class }}"
     no_entity: "Aucun appareil ou entité appelé {{ entity }}"
     handle_error: "Une erreur est intervenue pendant le traitement"
-lists: {}
+lists:
+  color:
+    values:
+      - in: "blanche"
+        out: "white"
+      - in: "noire"
+        out: "black"
+      - in: "rouge"
+        out: "red"
+      - in: "orange"
+        out: "orange"
+      - in: "jaune"
+        out: "yellow"
+      - in: "verte"
+        out: "green"
+      - in: "bleue"
+        out: "blue"
+      - in: "violette"
+        out: "purple"
+      - in: "marron"
+        out: "brown"
+      - in: "rose"
+        out: "pink"
+  brightness:
+    range:
+      type: "percentage"
+      from: 0
+      to: 100
+  temperature:
+    range:
+      type: "temperature"
+      from: 0
+      to: 100
 expansion_rules:
+  #name: "([le | la | les] {name}|[l']{name})"
   name: "([le | la | les] {name} | [l']{name})"
-  area: "([le | la] {area} | [l']{area})"
+  area: "([le | la | du] {area} | [l']{area})"
+  brightness: "{brightness:brightness_pct} [pourcent]"
+  what_is: "(qu'est ce que | qu'est-ce que)"
+  turn: "(interrupteur)"
+  temperature: "{temperature} [degré] [{temperature_unit}]"
+  set: "(règle | met)"
+  close: "(ferme)"
+  turn_on: "(allume | active | démarre)"
+  turn_off: "(éteint | éteins | désactive | stop | arrête)"
+  lights: "(la lumière | les lumières)"
+  bulbs: "(la lampe | les lampes)"
+  do: "(fais | fait)"
```

Comment:
The keys should be in French, not English. They should only be in English if they wrap a slot (like `{name}`)

Reply:
i have check and updated expension rules from english to french